app/org/sagebionetworks/bridge/services/AuthenticationServiceImpl.java

```diff
@@ -163,13 +163,12 @@ public void signUp(Study study, SignUp signUp, boolean isAnonSignUp) {
             }
             accountDao.signUp(study, signUp, isAnonSignUp);
         } catch(EntityAlreadyExistsException e) {
-            // Do not indicate to the end user that this email address has already been taken, as this 
-            // leaks information about who is in the study. Instead send a reset password request to the 
-            // email address in case user has forgotten password and is trying to sign up again.
+            // Suppress this. Otherwise it reveals if the account does not exist
+            // Non-anonymous sign ups (sign ups done by admins on behalf of users) still get a 404
             if (isAnonSignUp) {
-                logger.info(String.format("Sign up attempt for existing account: %s, %s", study.getIdentifier(), signUp.getEmail()));
                 Email email = new Email(study.getIdentifier(), signUp.getEmail());
                 requestResetPassword(study, email);
+                logger.info("Sign up attempt for existing account: "+study.getIdentifier()+" "+email.getEmail());
```

Comment:
Should we be logging email addresses at all? This is private customer information.